Resources/doc/index.rst

```diff
@@ -41,12 +41,11 @@ application:
 
     # config/packages/doctrine_migrations.yaml
 
-    doctrine_migrations:
```

Comment:
:thinking: why are you removing this? It still appears here: https://github.com/symfony/recipes/blob/0d06144589aeb8eff11025d398f724f9545fab2c/doctrine/doctrine-migrations-bundle/3.1/config/packages/doctrine_migrations.yaml#L1